### DISCUSSION TITLE
docker buildx failing intermittently due to unexpected commit digest

### DISCUSSION THREAD

**Discussion by ZahirJ** [docker/buildx#1872] [Q&A] [answered]
When we try to build our docker image (using multiple FROMs and add a war at the last step ) - last step is failing on sha256 mismatch from time to time for no good reason:

> docker buildx build --pull --push --platform linux/amd64,linux/arm64 --build-arg XXX_WAR_PATH=target/xxx/xxx.war --build-arg XXX_BASE_TAG=7.59.9 --build-arg XXX_DOCKER_IMAGE_NAME=xxx --build-arg XXX_DOCKER_CLEAN_IMAGE_TAG=7.59.x -t xxx.yyy.io/yyy/xxx -f Dockerfile-xxx .

`…..building layers …

`# Copying app into tomcat directory of the build image
 
`#10 [linux/arm64 2/5] COPY target/XXX.war /opt/yyy/XXX/app/XXX/tomcat/webapps
`#10 ERROR: failed commit on ref "layer-`sha256:00c56312925ecb176d62ab80b6723c8b4d3bde87fa80911c15b51866e1ff198c": unexpected commit digest `sha256:97550bc91c48be9344da955a072a612f69da22de67ccd7935252f44887618a04, expected `sha256:00c56312925ecb176d62ab80b6723c8b4d3bde87fa80911c15b51866e1ff198c: failed precondition

`#11 [linux/amd64 2/5] COPY target/xxx/xxx.war /opt/yyy/xxx/app/xxx/tomcat/webapps
`#11 CANCELED

` ERROR: failed to solve: failed to compute cache key: failed commit on ref "layer-`sha256:00c56312925ecb176d62ab80b6723c8b4d3bde87fa80911c15b51866e1ff198c": unexpected commit digest `sha256:97550bc91c48be9344da955a072a612f69da22de67ccd7935252f44887618a04, expected `sha256:00c56312925ecb176d62ab80b6723c8b4d3bde87fa80911c15b51866e1ff198c: failed precondition
`+ errorExit 'Failed building XXX Docker image!'

`ERROR: Failed building XXX Docker image!

`+ echo
`+ echo 'ERROR: Failed building XXXX Docker image!'
`+ echo
`+ exit 1

**Comment by jedevc** [comment, marked as answer]
Looks like this is a duplicate of https://github.com/docker/buildx/issues/1775, I'll close this in preference of that one :tada:
